imod/mf6/hfb.py

```diff
@@ -1105,4 +1120,4 @@ def from_imod5_dataset(cls, imod5_data: dict[str, dict[str, GridDataArray]]):
             dataframe_ls.append(geometry_layer)
         compound_dataframe = pd.concat(dataframe_ls, ignore_index=True)
 
-        return LayeredHorizontalFlowBarrierResistance(compound_dataframe)
+        return cls(compound_dataframe)
```

Comment:
there is also a "LayeredHorizontalFlowBarrierHydraulicCharacteristic". 
Should the changes in this PR also be applied there?

Reply:
I think not. The hydraulic characteristic is a MODFLOW 6 concept: it is the inverse of the resistance. In the end, both the ``Resistance`` as well as the ``HydraulicCharacteristic`` variant are converted to a hydraulic characteristic for the Modflow6 HFB package. iMOD5 only allows you to specify a hfb with a resistance, so I don't see a real use case for adding a ``from_imod5_data`` method to the HydraulicCharacteristic class.